commander/package-lock.json

```diff
@@ -119,6 +119,95 @@
 				"to-fast-properties": "^2.0.0"
 			}
 		},
+		"@liskhq/bignum": {
+			"version": "1.3.1",
+			"resolved": "https://registry.npmjs.org/@liskhq/bignum/-/bignum-1.3.1.tgz",
+			"integrity": "sha512-q9+NvqbpmXOqpPmV8Y+XSEIUJFMZDGyfW6rkN9Ej3nzPb/qurY/Ic2UPTeTTaj8+q/bcw5JUwTb86hi7PIziDg==",
+			"requires": {
+				"@types/node": "11.11.2"
+			},
+			"dependencies": {
+				"@types/node": {
+					"version": "11.11.2",
+					"resolved": "https://registry.npmjs.org/@types/node/-/node-11.11.2.tgz",
+					"integrity": "sha512-iEaHiDNkHv4Jrm9O5T37OYEUwjJesiyt6ZlhLFK0sbo4CLD0jyCOB4Pc2F9iD3MbW2397SLNxZKdDGntGaBjQQ=="
+				}
+			}
+		},
+		"@liskhq/lisk-api-client": {
```

Comment:
siblings package shouldn't be in the `package-lock`.
Please use 
`npx lerna boostrap` in the root level to update the `package-lock`

Reply:
@shuse2 i tried the `npx lerna boostrap` in the root and it dint resolve the package lock issue